codelabs/grpc-java-getting-started/start_here/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java

```diff
@@ -40,7 +39,7 @@ public void getFeature(int lat, int lon) {
 
     Point request = Point.newBuilder().setLatitude(lat).setLongitude(lon).build();
 
-    Feature feature;
+    Feature feature = null;
```

Comment:
It would be better to put `feature = null` in the `try` block so that it is next to the hint.

Reply:
Good idea!